build.sbt

```diff
@@ -204,6 +213,7 @@ lazy val monixStream = (project in file("monix-stream"))
       "org.typelevel" %% "cats-core" % catsVersion,
       "org.typelevel" %% "cats-effect" % catsEffectsVersion,
       "io.monix" %% "monix-eval" % monixVersion,
+      "io.monix" %% "monix-reactive" % monixVersion,
```

Comment:
Does this duplicate line 217 or it's a github glitch?

Reply:
I hope lol.

Will check tomorrow once I am on a computes again.

Reply:
Indeed it was duplicated, sorry about that!
Maybe it happened during the `merge main`?

Anyways, it strikes me that **sbt** doesn't identify that as an error, but at least it is fixed now!
Thanks!